hashes/src/lib.rs

```diff
@@ -205,13 +205,13 @@ pub trait Hash: Copy + Clone + PartialEq + Eq + PartialOrd + Ord +
     const DISPLAY_BACKWARD: bool = false;
 
     /// Unwraps the hash and returns the underlying byte array.
-    fn into_inner(self) -> Self::Inner;
+    fn to_byte_array(self) -> Self::Bytes;
 
     /// Unwraps the hash and returns a reference to the underlying byte array.
```

Comment:
Unrelated, but "Unwraps" sounds scary. I'd just write "Returns a reference to the underlying byte array". Same above.

Reply:
Sweet, I'll change it while we are here.